packages/server/src/authenticatorKey/parseNoneKey.js

```diff
@@ -0,0 +1,99 @@
+const {
+    hash,
+    convertASN1toPEM,
+    verifySignature,
+    convertCOSEPublicKeyToRawPKCSECDHAKey,
+  } = require('../utils');
+
+  exports.parseNoneKey = (authenticatorKey, clientDataJSON) => {
+    const authenticatorData = parseAttestationData(authenticatorKey.authData);
+
+    if (!authenticatorData.flags.up) {
+        throw new Error('User was NOT presented during authentication!');
+    }
+
+    const publicKey = convertCOSEPublicKeyToRawPKCSECDHAKey(
+        authenticatorData.COSEPublicKey
+    );
+
+    return {
+        fmt: 'none',
+        publicKey: publicKey.toString('base64'),
+        counter: authenticatorData.counter,
+        credID: authenticatorData.credID.toString('base64'),
+    };
+  };
+
+  exports.validateNoneKey = (
+    authenticatorDataBuffer,
+    key,
+    clientDataJSON,
+    base64Signature
+  ) => {
+    const authenticatorData = parseAttestationData(authenticatorDataBuffer);
+
+    if (!authenticatorData.flags.up) {
+        throw new Error('User was NOT presented durring authentication!');
```

Comment:
Nitpick: `during` typo.

Reply:
Yeah... that's copied from the other parts of the library. 😆 I figured I'd be consistent for now.